src/site/xdoc/server/config-domainlist.xml

```diff
@@ -33,6 +33,8 @@
       
       <p>Using Spring, you must specify the DomainList class as attribute of the domainlist tag (org.apache.james.domainlist.jpa.JPADomainList, or org.apache.james.domainlist.xml.XMLDomainList, or...). This have no effect when using Guice.</p>
 
+      <p>JAMES-Guice uses org.apache.james.domainlist.cassandra.CassandraDomainList by default and you don't need to specify class attribute in <strong>domainlist</strong> tag</p>
```

Comment:
Well James guice JPA don't uses the Cassandra domain list.

Maybe saying that guice packages the domain list appropriate to my backend is enough...

Reply:
I'm not sure about the wording :/ if you could do it, it'll be safer.

Reply:
`JAMES-Guice uses a bundled domain list and you don't need to specify class attribute in <strong>domainlist</strong> tag `